starboard/nplb/compiler_compliance/cpp17_support.cc

```diff
@@ -64,6 +64,28 @@ static_assert(add_one(1) == 2, "Constexpr lambdas support is required");
 static_assert(starboard::foo::bar::baz::life() == 42,
               "Nested Namespaces support is required");
 
+
+// Test fallthrough support
+
+constexpr int test_fallthrough(int n) {
+    switch (n)
+    {
+    case 1:
+        return 1;
+        break;
+    case 2:
+        [[fallthrough]]; // intentional fallthrough
+    case 3:
+        return 3;
+        break;
+    }
+
+    return 0;
+}
+
+static_assert (test_fallthrough(2) == 3, "fallthrough support is required");
```

Comment:
Perhaps `[[fallthrough]] attribute support is required`, on a rare instance when you'd see this compiler error

Reply:
done